virttest/qemu_vm.py

```diff
@@ -2302,11 +2293,17 @@ def sort_key(dev):
             # disable-legacy and disable-modern options explicitly.
             set_disable_legacy = params.get("set_disable_legacy", "no")
             set_disable_modern = params.get("set_disable_modern", "no")
+            set_iommu_platform = params.get("set_iommu_platform", "no")
+            set_ats = params.get("set_ats", "no")
             if dev_type in virtio_pci_devices:
                 if set_disable_legacy == "yes":
-                    add_disable_legacy(devices, device, dev_type)
+                    add_virtio_option("disable-legacy", "on",  devices, device, dev_type)
                 if set_disable_modern == "yes":
-                    add_disable_modern(devices, device, dev_type)
+                    add_virtio_option("disable-modern", "off", devices, device, dev_type)
```

Comment:
default value for disable-legacy should be on, right? refer line 1321

Reply:
@CongLi  same with disable_legacy, thanks !

Reply:
same as above.

Reply:
same as above :)